packages/app/src/cli/constants.ts

```diff
@@ -144,6 +149,8 @@ export function getUIExtensionRendererDependency(extensionType: UIExtensionTypes
   switch (extensionType) {
     case 'product_subscription':
       return {name: '@shopify/admin-ui-extensions-react', version: '^1.0.1'}
+    case 'checkout_ui_extension_beta':
+      return {name: '@shopify/checkout-ui-extensions-react', version: '^0.20.0'}
```

Comment:
Note that the new checkout extension differs from existing extension in that it has two rendering packages:

1. "@shopify/admin-ui-extensions"
2. "@shopify/checkout-ui-extensions"

To support this we'd need to update some code in the CLI to be able to return two render dependencies, e.g:

```
export function getUIExtensionRendererDependency(extensionType: UIExtensionTypes): DependencyVersion | undefined {
  switch (extensionType) {
    case 'checkout_ui_extension_beta':
      return {name: '@shopify/checkout-ui-extensions-react', version: '^0.20.0'}
    // ...etc
  }
}
```

Becomes
```
export function getUIExtensionRendererDependency(extensionType: UIExtensionTypes): DependencyVersion | undefined {
  switch (extensionType) {
    case 'checkout_ui_extension_beta':
      return [
        {name: '@shopify/checkout-ui-extensions-react', version: '^0.20.0'},
        {name: '@shopify/admin-ui-extensions-react', version: '^0.20.0'}
      ]
    // ...etc
  }
}
```

This becomes the dependencies we add to the apps package.json.

But! I _think_ both of these packages will become the same package... so can we just skip this for now?

Reply:
> Umm, if the idea is that no extension should have more than 1 dependency (I guess a common @shopify/ui-extension is the goal) I think we shouldn't change this: You'll be forced to update the code in more files to support an array here and is affecting the CLI codebase for internal work 

Yeah, agreed.  We'll leave this as is for now.

> Would it make more sense to do this in a private fork? That way you could hardcode any change you need without worrying about the CLI codebase for the moment.

My instinct is I'd prefer to merge into main so we don't have to deal with a bunch of merge conflicts later. WDYT?

Reply:
In theory we are going to start refactoring this next week, so instead of merging back and having conflicts we could just add support for the new extension type after the refactor. (while you work in your fork with this version)

I'm saying this in case you need to do more changes here, because then we will have conflicts during the refactor, but if you don't plan to do any other changes to this, then is ok merge it

Reply:
Ok, I'll use this as a feature branch then.  Mostly I'll use the feature branch to learn, and to have some code that we can copy to the new architecture.

This means that this PR will remain open and I'll merge into this branch.  Not ideal, but it's just unfortunate timing with the refactor... which is well worth it!